### ISSUE TITLE
Error compiling with OCaml 5.0

### ISSUE THREAD

**Issue by davisuga** [0xffea/ocaml-redis#89]
When I run opam install redis I get the following error:

```#=== ERROR while compiling redis.0.5 ==========================================#
# context     2.1.0 | linux/x86_64 | ocaml-base-compiler.5.0.0~alpha0 | https://opam.ocaml.org#366196f9
# path        ~/.opam/5.0.0~alpha0/.opam-switch/build/redis.0.5
# command     ~/.opam/opam-init/hooks/sandbox.sh build dune build -p redis -j 7
# exit-code   1
# env-file    ~/.opam/log/redis-847604-f1b4f6.env
# output-file ~/.opam/log/redis-847604-f1b4f6.out
### output ###
# [...]
# Use Uuidm.v instead
# (cd _build/default && /home/davi/.opam/5.0.0~alpha0/bin/ocamlopt.opt -w -40 -safe-string -warn-error -3-32-34 -g -I src/.redis.objs/byte -I src/.redis.objs/native -I /home/davi/.opam/5.0.0~alpha0/lib/ocaml/unix -I /home/davi/.opam/5.0.0~alpha0/lib/re -I /home/davi/.opam/5.0.0~alpha0/lib/re/str -I /home/davi/.opam/5.0.0~alpha0/lib/seq -I /home/davi/.opam/5.0.0~alpha0/lib/uuidm -intf-suffix .ml[...]
# File "src/client.ml", line 57, characters 20-38:
# 57 |       let compare = Pervasives.compare
#                          ^^^^^^^^^^^^^^^^^^
# Error: Unbound module Pervasives
# (cd _build/default && /home/davi/.opam/5.0.0~alpha0/bin/ocamlopt.opt -w -40 -safe-string -warn-error -3-32-34 -g -I src/.redis.objs/byte -I src/.redis.objs/native -I /home/davi/.opam/5.0.0~alpha0/lib/ocaml/unix -I /home/davi/.opam/5.0.0~alpha0/lib/re -I /home/davi/.opam/5.0.0~alpha0/lib/re/str -I /home/davi/.opam/5.0.0~alpha0/lib/seq -I /home/davi/.opam/5.0.0~alpha0/lib/uuidm -intf-suffix .ml[...]
# File "src/mutex.ml", line 39, characters 31-37:
# 39 |     let id = Uuidm.(to_string (create `V4)) in
#                                     ^^^^^^
# Alert deprecated: Uuidm.create
# Use Uuidm.v instead```

**Comment by c-cube** [comment]
Simplest fix would be to require 4.07 (I think it's the version where `Stdlib` was added), or use stdlib-shims.

**Comment by c-cube** [comment]
Actually, can't reproduce on master. What version are you using? The latest release doesn't seem to contain any mention of `Pervasives` and it's already using `stdlib-shims`?

Please reopen if it still fails on latest release.

**Comment by c-cube** [comment]
Good point actually, it was fixed after 0.5! But 0.6 was recently released and should contain the fix :)
